modules/tinymce/src/themes/silver/test/ts/headless/toolbar/SearchableMenuButtonTest.ts

```diff
@@ -0,0 +1,540 @@
+                  structSearchLeafItem({ selected: false }),
+                  structSearchLeafItem({ selected: false }),
+                  structSearchParentItem({ selected: false, expanded: false })
+                ]
+              })
+            ])
+          ]
+        });
+      }),
+      tmenu
+    );
+    await assertAriaConsistent();
+
+    // Now, press <up> to select last item
+    Keyboard.activeKeystroke(hook.root(), Keys.up(), { });
+    Assertions.assertStructure(
+      'Checking structure of mailmerge dropdown (after pressing <down>)',
+      ApproxStructure.build((s, _str, _arr) => {
+        return s.element('div', {
+          children: [
+            structMenuWith({ selected: false }, [
+              structSearchField(''),
+              structSearchResultsWith([
+                structSearchLeafItem({ selected: false }),
+                structSearchLeafItem({ selected: false }),
+                structSearchParentItem({ selected: false, expanded: false }),
+                structSearchParentItem({ selected: true, expanded: true })
+              ])
+            ]),
+            structMenuWith({ selected: true }, [
+              s.element('div', {
+                children: [
+                  structSearchLeafItem({ selected: false }),
+                  structSearchLeafItem({ selected: false }),
+                  structSearchLeafItem({ selected: false }),
+                  structSearchParentItem({ selected: true, expanded: false })
+                ]
+              })
+            ])
+          ]
+        });
+      }),
+      tmenu
+    );
+    await assertAriaConsistent();
+
+    // And press <enter> to open the submenu for this last item. We should now
+    // have three menus open.
+    Keyboard.activeKeystroke(hook.root(), Keys.enter(), { });
+    Assertions.assertStructure(
+      'Checking structure of mailmerge dropdown (after pressing <enter>)',
+      ApproxStructure.build((s, _str, _arr) => {
+        return s.element('div', {
+          children: [
+            structMenuWith({ selected: false }, [
+              structSearchField(''),
+              structSearchResultsWith([
+                structSearchLeafItem({ selected: false }),
+                structSearchLeafItem({ selected: false }),
+                structSearchParentItem({ selected: false, expanded: false }),
+                structSearchParentItem({ selected: true, expanded: true })
+              ])
+            ]),
+            structMenuWith({ selected: false }, [
+              s.element('div', {
+                children: [
+                  structSearchLeafItem({ selected: false }),
+                  structSearchLeafItem({ selected: false }),
+                  structSearchLeafItem({ selected: false }),
+                  // This expanded=false looks like a bug. The submenu is
+                  // still expanded! But our formatting menus are the same.
+                  // INVESTIGATE
```

Comment:
Is this still accurate? I thought you investigated it and we said it was a bug and you'd logged a Jira to fix that?

Reply:
- [ ] connect to JIRA